components/engine/engine-odata/src/test/java/org/eclipse/dirigible/components/odata/transformers/OData2ODataMTransformerTest.java

```diff
@@ -61,15 +61,15 @@ public void testTransformOrders() throws IOException, SQLException {
         OData definition = ODataSynchronizer.parseOData("/orders/Orders.odata", orders);
 
         Table model = new Table("ORDERS");
-        TableColumn column1 = new TableColumn("Id", "Edm.Int32", "0", true, true, model);
-        TableColumn column2 = new TableColumn("Customer", "Edm.String", "20", model);
+        new TableColumn("Id", "Edm.Int32", "0", true, true, model);
```

Comment:
Same comment about the side effects we may have to check

Reply:
fixed